book/content/code_styling/Code_Styling.md

```diff
@@ -0,0 +1,279 @@
+# Writing Clear and (Human) Readable Code
+
+## Introduction 
+
+Have you ever opened a syntax or script file two years after running an analysis only find that you have no memory of the code?
+On top of this you may have lots of variables being passed to arcane functions, which are in turn passed back to themselves:
+
+```
```

Comment:
```suggestion
```r
```
Usually with markdown, this can help us to highlight code in a particular way, if we know what language is being used. (Not actually sure if the book is doing syntax highlighting though)

[This link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting) provides more detail